spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplate.java

```diff
@@ -206,11 +206,9 @@ public Exchange getExchange(String vhost, String name) {
 	}
 
 	private List<Queue> converteQueueList(List<QueueInfo> queues) {
-		List<Queue> convertedQueues = new ArrayList<Queue>();
-		for (QueueInfo qi : queues) {
-			convertedQueues.add(convert(qi));
-		}
-		return convertedQueues;
+		return queues.stream()
+				.map(qi -> convert(qi))
```

Comment:
I'm fairly certain that you could replace all of these with method references. e.g.: `.map(this::convert))`.